test/k23si/K23SITest.cpp

```diff
@@ -587,6 +583,7 @@ seastar::future<> runScenario05() {
 int main(int argc, char** argv) {
     k2::App app("K23SITest");
     app.addOptions()("cpo", bpo::value<k2::String>(), "The endpoint of the CPO");
+    app.addOptions()("create_ddl", bpo::value<k2::ParseableDuration>(), "The duration to wait after called create partition");
```

Comment:
I'd describe this as "the duration to wait for collection to be created". Also an option that sounds like a command is very confusing for what it is expecting(duration). Something like: "cpo_timeout" or "cpo_create_collection_timeout" would be better